lib/r10k/source/git.rb

```diff
@@ -43,8 +43,9 @@ class R10K::Source::Git < R10K::Source::Base
   # @param basedir [String] The base directory where the generated environments will be created.
   # @param options [Hash] An additional set of options for this source.
   #
-  # @option options [Boolean] :prefix Whether to prefix the source name to the
-  #   environment directory names. Defaults to false.
+  # @option options [Boolean, String] :prefix A String value to prefix the environment directory name.
```

Comment:
Ok.  I can't remember what I originally had,  I had changed this from previous feedback.  I'll clean it up. 

The same sentence is copied in these files: 
* environment/branchname.rb (after it's moved)
* source/base.rb
* source/git.rb  (err... uh.. i'll remove the docs from this class.    Honestly this is some of the first ruby code I've ever written so I'm not sure how the rdocs will behave but this might end up in duplicate documentation if it's also in the base class?)